src/schema/v2/bank_account.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLFieldConfig,
+} from "graphql"
+import { InternalIDFields } from "schema/v2/object_identification"
+import { ResolverContext } from "types/graphql"
+
+// fields: https://github.com/artsy/gravity/blob/main/db/schema.rb
+export const BankAccountType = new GraphQLObjectType<any, ResolverContext>({
+  name: "BankAccount",
+  fields: () => ({
+    ...InternalIDFields,
+    bankName: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Bank name",
+      resolve: ({ bank_name }) => bank_name,
+    },
```

Comment:
Defined a minimal bank account type - we should be clear on which fields need to be added (unless it is all of them).